🐛npm package is not installable

## Description

The bpmn-studio npm package is not installable.

```
> npm rebuild && electron-builder install-app-deps

sh: electron-builder: command not found
npm WARN sequelize-typescript@1.0.0-beta.3 requires a peer of @types/bluebird@* but none is installed. You must install peer dependencies yourself.
npm WARN sequelize-typescript@1.0.0-beta.3 requires a peer of @types/validator@* but none is installed. You must install peer dependencies yourself.

npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! bpmn-studio@5.6.2 postinstall: `npm rebuild && electron-builder install-app-deps`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the bpmn-studio@5.6.2 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
## Reproduction

`npm i -g bpmn-studio`

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   "main": "electron_app/electron.js",
   "scripts": {
     "start": "./bin/bpmn-studio.js",
-    "postinstall": "npm rebuild && electron-builder install-app-deps",
+    "postinstall": "npm rebuild",
     "start_dev": "au run --watch",
     "build": "au build",
     "reset": "bash dev_tooling/scripts/reset_setup.bash",
@@ -25,7 +25,7 @@
     "electron-build-windows": "npm run electron-prepare && build --config build/electron-builder.yml --publish never --windows && npm run electron-cleanup",
     "electron-build-macos": "npm run electron-prepare && build --config build/electron-builder.yml --publish never --macos && npm run electron-cleanup",
     "electron-cleanup": "node electron-builder/reset-name-in-package-json.js",
-    "electron-prepare": "node electron-builder/set-name-in-package-json.js && electron-rebuild && npm run build && node electron-builder/prepare-electron-build.js",
+    "electron-prepare": "electron-builder install-app-deps && node electron-builder/set-name-in-package-json.js && electron-rebuild && npm run build && node electron-builder/prepare-electron-build.js",
     "electron-start-dev": "npm run build && electron electron_app/electron.js",
     "jenkins-electron-install-app-deps": "electron-builder install-app-deps",
     "jenkins-electron-rebuild-native": "electron-rebuild --force",

